baselines/acer/acer.py

```diff
@@ -75,8 +75,8 @@ def __init__(self, policy, ob_space, ac_space, nenvs, nsteps, ent_coef, q_coef,
         train_ob_placeholder = tf.placeholder(dtype=ob_space.dtype, shape=(nenvs*(nsteps+1),) + ob_space.shape)
         with tf.variable_scope('acer_model', reuse=tf.AUTO_REUSE):
 
-            step_model = policy(observ_placeholder=step_ob_placeholder, sess=sess)
-            train_model = policy(observ_placeholder=train_ob_placeholder, sess=sess)
+            step_model = policy(nbatch=nenvs, nsteps=nsteps, observ_placeholder=step_ob_placeholder, sess=sess)
```

Comment:
Almost agree :) nsteps should be 1

Reply:
Whoops! :) sorry I missed that, just updated!